data_steward/utils/pipeline_logging.py

```diff
@@ -111,3 +159,80 @@ def setup_logger(log_filepath_list, console_logging=True):
     for filename in log_path:
         log_list.append(create_logger(filename, console_logging))
     return log_list
+
+
+def get_date_str():
+    return datetime.today().strftime(FILENAME_FMT)
+
+
+def get_log_file_path():
+    """
+    Get the abs path of the log file to use. 
+    
+    The location is DEFAULT_LOG_DIR and the file name is the current
+    date formatted using FILENAME_FMT.
+
+    :return: absolute path to the log file
+    """
+    date_str = get_date_str()
+    return os.path.join(DEFAULT_LOG_DIR, f'{date_str}.log')
+
+
+def get_config(filename, level):
+    """
+    Get a dictionary which describes the logging configuration
+    
+    :param filename: Create the FileHandler using the specified filename.
+    :param level: Set the root logger level to the specified level.
+    :return: the default configuration dict
+    """
+    # copy _DEFAULT_CONFIG to avoid modifying it
+    default_config = dict(_DEFAULT_CONFIG)
+    default_config['root']['level'] = level
```

Comment:
I assume level is one of the `logging.[INFO, DEBUG, WARNING, ERROR]` levels.  You should make this clear.  I don't expect the others to pick up on this on their own.  It wouldn't hurt to mention the possible enumerated values in the function doc string either.  Also, what about the case where someone didn't provide a level?  Is there a default value or default behavior?

Reply:
I'll add more detailed doc and also make everything except configure private